official/vision/image_classification/resnet/common.py

```diff
@@ -331,6 +331,14 @@ def define_pruning_flags():
   flags.DEFINE_integer('pruning_frequency', 100,
                        'Frequency for pruning.')
 
+def define_clustering_flags():
+  """Define flags for clustering methods."""
+  flags.DEFINE_string('clustering_method', None,
+                      'None (no clustering) or selective_clustering.')
+  flags.DEFINE_integer('number_of_clusters', 256,
+                       'Number of clusters used in each layer.')
+  flags.DEFINE_string('save_files_to', None,
```

Comment:
Don't think a new `save_files_to` flag should exist that's specific to clustering. There is existing export logic in the script already. The current code doesn't create a TFLite model and that seems fine.

Reply:
yes, that makes sense. Thanks.